fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -128,6 +129,16 @@ submitTaskStatement
 
 // ------------------------------------------- Materialized View Statement ---------------------------------------------
 
+createMaterializedViewStatement
+    : CREATE MATERIALIZED VIEW (IF NOT EXISTS)? mvName=qualifiedName
+    comment?
+    (PARTITION BY primaryExpression)?
```

Comment:
why the name is primaryExpression?

Reply:
We need expression like `identifier(.identifier)? `or `function(xx)`, i think `primaryExpression` has less matching scope than `Expression`.